static/js/webrtc.js

```diff
@@ -234,22 +240,38 @@ var rtc = (function() {
           videoContainer.addClass('local-user');
           video.muted = true;
         }
-        self.addInterface(userId);
+        self.addInterface(userId, stream);
       }
       if (stream) {
         attachMediaStream(video, stream);
       } else if (video) {
         $(video).parent().remove();
       }
     },
-    addInterface: function(userId) {
+    addInterface: function(userId, stream) {
       var isLocal = userId == self.getUserId();
       var videoId = "video_" + userId.replace(/\./g, "_");
       var $video = $("#" + videoId);
 
+      ///////
+      // Mute button
+      ///////
+
+      var audioTrack = stream.getAudioTracks()[0];
+      var initiallyMuted = true; // if there's no audio track, it's muted
+      if (audioTrack) {
```

Comment:
@JohnMcLear I've seen this test around this code before. Do you know of any _legitimate_ reason why `audioTrack` might be missing other than this new feature that hard-disables it? If not, I might be able to clean up the code a bit by treating it as an error earlier on and just assume it'll be here otherwise.

Reply:
I think it can probably be safely removed, one to do in a separate PR tho!